src/components/Profile/Collected.tsx

```diff
@@ -32,6 +33,7 @@ const Collected = () => {
             }),
           )
           if (result.data && result.data?.length > 0) {
+            pageView(`${router.asPath}/${fetchOffset}`)
```

Comment:
```suggestion
            pageView(`${router.asPath}/?page=${fetchOffset}`)
```

its probably better if we do it as a url param, that way in the future we might actually allow people to accessing those routes to visit that page